apps/web/src/components/item-review/index.tsx

```diff
@@ -26,7 +27,11 @@ function ItemReviewContent() {
       review={data.review}
     >
       <Button size="sm" variant="outline">
-        <Star size={14} className="mr-2" />
+        <Star
+          size={14}
+          fill={data.review ? '#eab308' : 'none'} // #eab308 is the tailwind color for yellow-500
+          className={cn(data.review && 'text-yellow-500', 'mr-2')}
```

Comment:
We are not using yellow as the review color, but rather `amber`

```suggestion
          className={cn('mr-2', data.review && 'fill-amber-500 text-amber-500')}
```